docs/lib/storage/fragments/common/overview.md

```diff
@@ -0,0 +1,21 @@
+## Storage with Amplify
+
+AWS Amplify Storage module provides a simple mechanism for managing user content for your app in public, protected or private storage buckets. The Storage category comes with built-in support for [Amazon S3 (Simple Storage Service)](https://docs.aws.amazon.com/AmazonS3/latest/dev/Welcome.html). 
```

Comment:
are we using uppercase "Storage" or lower case "storage"?

Reply:
lowercase. thanks for catching